lib_common.gradle

```diff
@@ -235,14 +235,7 @@ def getDependencyVersion(String semanticVersion) {
 }
 
 def getPublishUrl() {
-    switch (BuildType.fromBranch(currentBranch())) {
-        case BuildType.RELEASE:
-            return "http://packages.geotracinternational.com/nexus/repository/maven-releases"
-        case BuildType.BETA:
-            return "http://packages.geotracinternational.com/nexus/repository/maven-snapshots"
-        default:
-            throw new RuntimeException("Do NOT publish the library under developement")

```

Comment:
Part of this might make sense to keep. I'm not sure that it ever makes sense to allow someone to publish from HEAD, but from any other development branch, it should be fine.